src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -267,6 +267,12 @@ internal static object GetClaimValueUsingValueType(Claim claim)
                     return longValue;
             }
 
+            if (claim.ValueType == ClaimValueTypes.DateTime)
+            {
+                if (DateTime.TryParse(claim.Value, out DateTime dateTimeValue))
```

Comment:
what happens if TryParse fails?

Reply:
claim.Value will be returned, same as for the other types where parsing might fail.